### PR TITLE
cuetools: symlink cuetag to cuetag.sh

### PR DESCRIPTION
**Pull request by hhm0**
Debian-based distros call `cuetag.sh` "cuetag" (see https://packages.debian.org/buster/amd64/cuetools/filelist).
This adds a symlink for compatibility with those distros.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/cd-dvd/cuetools/default.nix [hunk 1]
```diff
@@ -15,6 +15,10 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ bison flac flex id3v2 vorbis-tools ];
 
+  postInstall = ''
```
Comment (worldofpeace) [inline pkgs/tools/cd-dvd/cuetools/default.nix]: Maybe a comment here that it's for compat with other distros?

Comment (hhm0) [reply pkgs/tools/cd-dvd/cuetools/default.nix]: added, thanks for the suggestion!

**Comment by worldofpeace** [comment]
@hhm0 Can you squash https://github.com/NixOS/nixpkgs/pull/56397/commits/c9c52b710484e41b44bcfa6c28e3300186c71591 and https://github.com/NixOS/nixpkgs/pull/56397/commits/ea13e3769ff4b2da25522de7468f002e149340c2?
( i don't like force pushing to people branches )

**Comment by hhm0** [comment]
@worldofpeace done :-)

**Comment by hhm0** [comment]
@worldofpeace Thanks! :-)

**Comment by worldofpeace** [comment]
@hhm0 Thank you :smile:

**Pull request merged**